bound/typescript/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "tbdex",
+  "name": "@tbdex/sdk",
```

Comment:
Thoughts on just `tbdex`? Looks like npm doesn't support `@tbdex` because the `@` is a scope which must be followed by package name, but we could be daring and just do `tbdex`, right?